camel/agents/chat_agent.py

```diff
@@ -303,6 +309,10 @@ def init_messages(self) -> None:
         self.memory.clear()
         self.memory.write_record(system_record)
 
+        # Write input memory records back if they exist
+        if self.input_memory_records:
+            self.memory.write_records(self.input_memory_records)
```

Comment:
If there exists a system message inside the memory (for example, the memory was used by another agent). This implementation will cause two system messages in one agent. May not be what we want

Reply:
updated